Gemfile

```diff
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
 
-ruby '>= 2.5', '< 2.6'
```

Comment:
I'm not sure this is sensible. The Ruby version that we're allowed to use is bound to the one available in the Platform.sh pipeline, which is in turn bound to whatever is available inside their PHP image.

Maybe we can increase the Ruby version by upgrading the PHP version?

Reply:
The non-PHP language runtimes should not be tied to PHP version. Perhaps I am misunderstanding your point though. This does not prevent running with 2.5 as the lower bound is correct. This merely enables running with a higher upper-bound (tested working). The docker for example still uses ruby 2.5 (also the proof-reading it checks for reports numerous failures, but perhaps that can be solved separately).

Reply:
The Ruby runtime is unfortunately tied due to how the deploy pipeline works: with a single container. So, inside that single container, we require a specific PHP version, and inside we find another, specific, not-chosen-by-us Ruby version. That's why I was saying that upgrading PHP may lead to upgrading Ruby.

Hence, running with a different Ruby version locally is not good IMHO, because it 's different from what will happen during the deploy.

Reply:
So the ruby you are bound to use won't change from this line is I guess the confusion I am having. This allows the same lower and upper bound range of ruby runtimes, but additionally, adds the flexibility to not rely on (deprecated, end-of-life) ruby 2.6 [source[(https://endoflife.date/ruby)

I understand a little about how a box might have only a single runtime. So running this on that box should not break. If it does, please can you share the message?

Reply:
My concern is that you may start having issues in the pipeline in PSH while deploying, while being unaware of that during local development, since you would be using a different version.

I know that it's EOL, and I'm tolerating it just because we're not actively deploying it, we're just using it to compile CSS and check links... But, in the end, I would be pretty happy to upgrade to PHP 8.1 and obtain a bump on Ruby too.

Reply:
Any progress on sharing failure messages? I'm more or less stuck on fixing without.